packages/backend/src/config/app.ts

```diff
@@ -42,7 +42,9 @@ export const Config = {
   openTelemetryCollectors: envStringArray(
     'OPEN_TELEMETRY_COLLECTOR_URLS',
     process.env.LIVENET
```

Comment:
```suggestion
    envBool(process.env.LIVENET)
```
I _think_. Otherwise, this would work if `LIVENET="false"`

Reply:
oh, yes. obviously. Thanks!